generator/python/build.gradle

```diff
@@ -60,40 +62,50 @@ task generateClient(type: GradleBuild) {
         
         def artifactVersion = isPreviewVersion ? '0' : criteoApiVersion.replaceAll("_",".").toString()
         artifactVersion += ".${generatorVersion}.${getFormattedDate()}".toString() // example : 2021.01.0.211110 for stable , 0.0.211110 for preview
+		
+		def createConfig  = task("createConfig_" + taskSuffix, type: Copy)
```

Comment:
fix indentation

Reply:
space and whitespaces mix can be indented as different on some views, All be aligned on spaces in a future commit